{{ cookiecutter.project_slug }}/docker/Dockerfile

```diff
@@ -1,14 +1,14 @@
 {%- if cookiecutter.use_nvidia_docker == 'yes' -%}
-FROM nvidia/cuda:8.0-cudnn5-devel-ubuntu16.04
+FROM nvidia/cuda:10.1-cudnn7-devel-ubuntu18.04
```

Comment:
This is the latest version of CUDA and CuDNN on https://hub.docker.com/r/nvidia/cuda/